.github/workflows/pkg-fedora.yml

```diff
@@ -26,8 +26,8 @@ jobs:
         # descriptive name, and one key 'containerid' with the name of the
         # container (i.e., what you want to docker-pull)
         include:
-          - distro: 'Fedora 33'
-            containerid: 'gnuradio/ci:fedora-33-3.9'
+          - distro: 'Fedora 34'
+            containerid: 'gnuradio/ci-fedora-34-3.9:1.0'
```

Comment:
```suggestion
          - distro: 'Fedora 33'
            containerid: 'gnuradio/ci:fedora-33-3.9'
          - distro: 'Fedora 34'
            containerid: 'gnuradio/ci:fedora-34-3.9' 
```

Reply:
These tags do not exist, so sadly not possible :(

Reply:
They do now.